system/t06_publish/PublishSwitch12Test_gold

```diff
@@ -1,8 +1 @@
-Loading packages...
-Generating metadata files and linking package files...
-Finalizing metadata files...
-Signing file 'Release' with gpg, please enter your passphrase when prompted:
-Clearsigning file 'Release' with gpg, please enter your passphrase when prompted:
-Cleaning up prefix "." components a, c...
-
-Publish for snapshot ./maverick [i386] publishes {a: [snap2]: Created as empty}, {b: [snap2]: Created as empty}, {c: [snap3]: Created as empty} has been successfully switched to new snapshot.
+ERROR: unable to switch: component c does not exist in published repository
```

Comment:
this is on purpose failing now ?

Reply:
Yes, I have reverted `switch`, so that the command has the original semantics again. That means you can only change snapshots of existing components and cannot add new components. The test must fail.